### ISSUE TITLE
Lower barriers-to-entry: A lesson from Wikipedia

### ISSUE THREAD

**Issue by jonkerz** [calacademy-research/antcat#129]
Not so much a proper issue but a compilation of general ideas that can be referred to from other issues. If you can edit this comment, please fix typos / improve grammar / improve the content. 

### Wikipedia - The Free Encyclopedia
As you probably know, anyone can edit nearly any Wikipedia article, including Featured Articles (which are considered to be the best articles Wikipedia has to offer) -- even without logging in. This means the community has to spend time reverting vandalism. "[A]bout 97% of vandalism comes from anonymous users" ([source](https://en.wikipedia.org/wiki/Wikipedia:Perennial_proposals#Prohibit_anonymous_users_from_editing)).

Forcing visitors to register an account to edit would make vandalism drop by orders of magnitudes. However, that approach would effectively kill the stream of new users. A minority of the community even wants to go as far as forcing users to use their real names; if enforcing user registration wouldn't make sure that no visitors started to contribute to the 'Pedia, that policy certainly would. Seeing your change published for the world to see gives an instant gratification, and we do not want to deny users that. It's nice to be nice to people, but from a more practical (or "Machiavellian") point-of-view: we want users to enjoy ("become addicted to") editing, and to make editing as easy as possible, in order for first-time contributors to be more likely to become regular editors.

**TL;DR**: Anyone can edit --> some visitors give try it --> some of them become regular users (and some of them become highly active users) --> better encyclopedia --> makes the Wikipedia people happy.

### AntCat - An Online Catalog of the Ants of the World
Allowing anyone to edit the catalog is never going to happen*, but we can:
* Make account registration open (more new users).
* Make it easier for visitors to contribute errata (more feedback/corrections, and some of the submitters will become new users, and some of them editors).
* Improve the editing interface to make it more user-friendly (so that editors are more likely to edit -- committing a change that you have no idea how it affects the database and you don't even know if it can be reversed, is scary)
* Make existing users feel "more a part of the community" and let them "take more ownership" of the project by _eg_ enabling basic user profiles, or allowing non-editors to submit suggestion to editors via an open task feature (which means, of course, more edits!).
* Create new features such as a watchlist where users can add taxa they want to get notified about when they change (so that both hobbyist and experts are more likely to contribute to the project)

*allowing anyone to edit the catalog, but only publishing approved edits would on the other hand be both possible and a very nice feature, but we're not there yet.

**TL;DR**: A more open community and easier to edit --> more users and more edits --> better catalog --> makes the ant people happy.

### Other key features of Wikipedia, and why we want the same
* On Wikipedia: All changes are reversible and logged -- messing up really isn't a big deal.
  * What we want for AntCat: Ditto.
  * Concrete suggestions: Activity feed; history/change browser.
* On Wikipedia: In order to create great articles, users must be allowed to write great articles.
  * What we want for AntCat: More open community; make it easier to become an editor.
  * Concrete suggestions: Make registration open; allow non-editors to add open tasks, invite active users to become editors.
* On Wikipedia: Editing a wiki is inherently collaborative; all activity is public.
  * What we want for AntCat: More collaboration.
  * Concrete suggestions: Open task feature, activity feed.

**TL;DR**: Wikipedia is great, we want more of that.

Note: On Wikipedia, anyone with an account is an "editor" or "user" (but more generally it also includes anyone who edits). On AntCat, "editor" or "user" generally means someone who has been approved to edit the catalog (but "user" could also mean anyone with an account). On both projects, "user" could also refer to a "visitor".

**Comment by bpescador** [comment]
@jonkerz This is a very well written and compelling summary of the changes needed for AntCa to address the social aspect of a community supported tool.  I agree that Wikipedia is a great model on how to engage the community.  AntCat is **not** sustainable unless we find a way to engage the community. 

I would also add that we need to be sure AntCat has protocols for all types of taxonomic changes.  For now, I am not clear on how the genus level changes should be made.
